packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -174,6 +174,11 @@ BuildApp() {
       EchoError "========================================================================"
       exit -1
     fi
+
+    local bitcode_flag=""
```

Comment:
Can you move this into the `if [[ -n "$LOCAL_ENGINE" ]];` block since it currently will fail without a local engine built with bitcode?  Since ENABLE_BITCODE is the default in Xcode I'm afraid someone in an add-to-app trying Flutter out for the first time will hit a bunch of weird errors if they don't swap it to NO.
Of course it doesn't work now if a dev keeps the default YES, but the "local-engine required"  error is more confusing for someone just messing around with Flutter in their app the first time.

Reply:
I think this makes sense - what I was hoping here is that this would prompt people interested in using it to see how they can do it, but that's probably the wrong way right now.  Eventually we could move this back out.